bindata/network/additional-networks/crd/001-crd.yaml

```diff
@@ -13,3 +13,68 @@ spec:
     kind: NetworkAttachmentDefinition
     shortNames:
     - net-attach-def
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: ippools.whereabouts.cni.k8s.io
+spec:
+  group: whereabouts.cni.k8s.io
```

Comment:
If whereabouts isn't an official Kubernetes project then it shouldn't be using a `k8s.io` API group...

Reply:
Thanks for the eyes, this is an upstream oversight, it was intended to be cni.cncf.io.... I'll try to get it patched up...